_posts/2015-05-12-new-lipsum.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blog
+category: blog
+published: true
+title: New Lipsum
+---
+
+## A New Post
+
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent mollis, ante
```

Comment:
@mylifeisshan if we use better tools for markdown then we can comment on a line, instead of on a whole paragraph. I realize this doesn't have the exact commenting but it's a lot better.